<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptimiserSelectWindow</class>
 <widget class="QDialog" name="OptimiserSelectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="postCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Post Detection Parameters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ballCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ball Detection Parameters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="obstCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Obstacle Detection Parameters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lineCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Detection Parameters</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="genCB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>General Parameters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptimiserSelectWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptimiserSelectWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
